scripts/akrtma3d/exakrtma3d_rhist.ksh

```diff
@@ -45,105 +68,148 @@ rhcycle=t${rhcyc}z
 hpssdir2=${HPSSOUT}/2year/rh${year}/${yearmo}/$yrmoday
 hpssdir0=${HPSSOUT}/rh${year}/${yearmo}/$yrmoday
 
-#Redefine cyclist to save single analysis files MPondeca/30Jul2017
+#
+#   Get a listing of all files in the directory to be tarred
+#   and break the file list up into groups of files.
+#   Each list of files names the contents of its associated tar file.
+#   Then cd to the directory to be tarred.
+# 
+#cyclist=t25z
+#if [ $rhcyc = "00" ]
+#then
+#   cyclist="t00z|t01z|t02z|t03z|t04z|t05z"
+#   endcyc=05
+#elif [ $rhcyc = "06" ]
+#then
+#   cyclist="t06z|t07z|t08z|t09z|t10z|t11z"
+#   endcyc=11
+#elif [ $rhcyc = "12" ]
+#then
+#   cyclist="t12z|t13z|t14z|t15z|t16z|t17z"
+#   endcyc=17
+#elif [ $rhcyc = "18" ]
+##then
+ #  cyclist="t18z|t19z|t20z|t21z|t22z|t23z"
+ #  endcyc=23
+#fi
+
+   #Redefine cyclist to save single analysis files MPondeca/30Jul2017
    cyclist="${rhcyc}"                            #MPondeca/30Jul2017
 
 cd $DATA
 
-cp rtma3d.${cyclem1}.* $DATA/$PDYm1
-cp rtma3d.${cyclem2}.* $DATA/$PDYm2
-cp rtma3d.${cyclem3}.* $DATA/$PDYm3
+cp $COMINm1/rtma3d.${cyclem1}.* $DATA/$PDYm1
```

Comment:
These 3 lines were causing an error in the rhist output for Alaska. I'm not sure, though, if this was due to the missing COMINm1/2/3.  Note: There are some files for Alaska that are named as "rtma3d" (howv/gust) that will eventually need to be renamed with the correct NET. Perhaps the best way to handle this is to copy over all files with rtma3d or akrtma3d.